docker-compose.yml

```diff
@@ -100,7 +100,7 @@ services:
     links:
       - kafka
   db:
-    image: docker.io/debezium/postgres
+    image: docker.io/debezium/postgres:14-alpine
```

Comment:
Would it make sense to  you try using the Clowder PostgreSQL 14 image ? 

quay.io/cloudservices/postgresql-rds:14-1

Reply:
As far as I understand, this is a debezium-flavored PG with some extras that we need in development :thinking:

Reply:
```
This image is based upon [postgres:11-apline](https://hub.docker.com/_/postgres/) and adds two logical decoding plug-ins:

[postgres-decoderbufs](https://github.com/debezium/), based on Protocol Buffers and maintained by the Debezium community
[wal2json](https://github.com/eulerto/wal2json), based on JSON and maintained by the wal2json community
```

Reply:
https://github.com/RedHatInsights/ros-backend/pull/267#discussion_r1022626729